commands/command_clone.go

```diff
@@ -79,8 +81,36 @@ func cloneCommand(cmd *cobra.Command, args []string) {
 		fetchRef("HEAD", include, exclude)
 	} else {
 		pull(include, exclude)
+
+		err := postCloneSubmodules(args)
+		if err != nil {
+			Exit("Error performing 'git lfs pull' for submodules: %v", err)
+		}
+	}
+
+}
+
+func postCloneSubmodules(args []string) error {
+	// In git 2.9+ the filter option will have been passed through to submodules
+	// So we need to lfs pull inside each, if the original clone command
```

Comment:
Bah, stuff moved around and left this hanging, it's covered below now so I'll trim it.